README.md

```diff
@@ -109,7 +112,19 @@ var s3Adapter = new S3Adapter('accessKey',
                     directAccess: false,
                     baseUrl: 'http://images.example.com',
                     signatureVersion: 'v4',
-                    globalCacheControl: 'public, max-age=86400'  // 24 hrs Cache-Control.
+                    globalCacheControl: 'public, max-age=86400',  // 24 hrs Cache-Control.
+                    validateFilename: (filename) => {
+                      if (filename.length > 1024) {
+                         return new Parse.Error(
+                            Parse.Error.INVALID_FILE_NAME,
```

Comment:
@dplewis Hmm... this might be problematic?

Reply:
No longer necessary: https://github.com/parse-community/parse-server/pull/6246

Reply:
You should `throw new Parse.Error` instead of returning

Reply:
Actually ignore that last comment this is correct.

Reply:
@dplewis @acinader So are we good to go here?